app/views/users/edit_security.html.erb

```diff
@@ -1,183 +1,199 @@
+        <span class="h2 bold">Sessions</span>
+        <%= badge_for @all_sessions.count, class: "bg-muted" %>
+      </legend>
+      <%= link_to logout_all_user_path(@user), method: :delete,
+              class: "btn bg-error py1 px2 h4", disabled: @sessions.count <= (current_user == @user ? 1 : 0) || disabled do %>
+        <%= inline_icon "door-leave", size: home_action_size %>
+        Sign out everywhere else
       <% end %>
     </div>
-  <% end %>
-  <div class="flex items-center justify-between flex-wrap gap-x-5 gap-y-2 mt3 pb2">
-    <legend class="flex flex-row justify-start items-center">
-      <span class="h2 bold">Sessions</span>
-      <%= badge_for @all_sessions.count, class: "bg-muted" %>
-    </legend>
-    <%= link_to logout_all_user_path(@user), method: :delete,
-            class: "btn bg-error py1 px2 h4", disabled: @sessions.count <= (current_user == @user ? 1 : 0) do %>
-      <%= inline_icon "door-leave", size: home_action_size %>
-      Sign out everywhere else
-    <% end %>
-  </div>
-
-  <%= form_with(model: @user, local: true, class: "pb2", data: { turbo_frame: "_top" }) do |form| %>
-    <div class="field">
-      <%= form.label :session_validity_preference, "Sign me out after" %>
-      <%= form.select :session_validity_preference, SessionsHelper::SESSION_DURATION_OPTIONS.to_a, selected: @user.session_validity_preference %>
-    </div>
 
-    <div class="submit">
-      <%= form.submit "Save" %>
-    </div>
-  <% end %>
+    <%= form_with(model: @user, local: true, class: "pb2", data: { turbo_frame: "_top" }) do |form| %>
+      <div class="field">
+        <%= form.label :session_validity_preference, "Sign me out after" %>
+        <%= form.select :session_validity_preference, SessionsHelper::SESSION_DURATION_OPTIONS.to_a, {}, selected: @user.session_validity_preference, disabled: %>
+      </div>
 
-  <div class="grid grid--split">
-    <% @all_sessions.each do |session| %>
-      <% if session.is_a? UserSession %>
-        <%= render "user_session", session: %>
-      <% elsif session.respond_to? :authorization_count %>
-        <%= render "oauth_authorization", authorization: session %>
-      <% end %>
+      <div class="submit">
+        <%= form.submit "Save", disabled: %>
+      </div>
     <% end %>
-  </div>
 
-  <% if @expired_sessions.any? %>
-    <h3 class="inline-flex items-center">
-      Recently logged out
-      <%= badge_for @expired_sessions.count, class: "bg-muted" %>
-    </h3>
     <div class="grid grid--split">
-      <% @expired_sessions.each do |session| %>
+      <% @all_sessions.each do |session| %>
         <% if session.is_a? UserSession %>
-          <%= render "user_session", session: %>
+          <%= render "user_session", session:, disabled: %>
         <% elsif session.respond_to? :authorization_count %>
-          <%= render "oauth_authorization", authorization: session %>
+          <%= render "oauth_authorization", authorization: session, disabled: %>
```

Comment:
```suggestion
          <%= render "oauth_authorization", authorization: session %>
```

It appears `oauth_authorization` is not using it at the moment. we'll add it in another PR.